### ISSUE TITLE
Big Sur (macOS 11) compatilibity

### ISSUE THREAD

**Issue by borekb** [dteoh/SlowQuitApps#82]
After an upgrade to macOS 11.0.1 Big Sur, I no longer see the SlowQuitApps behavior – Cmd-Q quits apps immediately.

Can anyone confirm that it works for them? Do I need to reinstall or something?

**Comment by dteoh** [comment]
I'm sorry but it is likely that Big Sur will not be supported anytime soon.

**Comment by borekb** [comment]
Is it because something fundamentally changed in the OS and apps like SlowQuitApps can no longer work?

**Comment by dteoh** [comment]
I'm sure that it can be made to work :)

It is a matter of putting in the time to do the necessary investigative and implementation work. I haven't really been active in this project for quite some time, and I haven't used the app regularly for a while now, so there is not much motivation for me to spend my free time on this.

Pull requests are always welcome :)

**Comment by borekb** [comment]
Got it, thanks, totally understandable 😄.

**Comment by nickimola** [comment]
Hi guys

Just to reassure you, slowquitapps works perfectly fine on big sure on both my macs.
One is 2018 MBP 13" that I use for work, so I installed BigSur as an update without formatting

The other one is a MBP 16" mid2014 and I installed BigSur after formatting

IN both cases installed slowquitapps from brew and slowquitapps works perfectly fine on both my machines :)

**Comment by borekb** [comment]
@nickimola interesting, I did an OS upgrade and tried to uncheck and re-check the permissions for SlowQuitApps but that wasn't enough. Will try uninstalling and reinstalling..

**Comment by borekb** [comment]
Alright, all works well after a full reinstall. Probably the step of **completely removing** SlowQuitApps from the Privacy → Accessibility list and adding it back is what I missed earlier – my bad.

Thanks for a great utility, @dteoh!

**Comment by gkatsev** [comment]
Just wanted to post another note to say that re-installing and removing and re-adding SlowQuitApps in the Accessibility permissions made it work for me on Big Sur as well.

**Comment by borekb** [comment]
I've just upgraded to a new M1 MBP / macOS 12 Monterey and am trying the usual procedure of removing / re-adding SlowQuitApps but no luck this time: it just doesn't work.

I've also tried uninstalling and re-installing the app, `killall SlowQuitApps`, restarting the computer and other things but it's still the same and doesn't work.

This is what I see after every app launch:

<img width="549" alt="Screen Shot 2021-11-16 at 14 50 06" src="https://user-images.githubusercontent.com/101152/141997460-05291139-79bf-4c48-88bb-8c6e8d7bdd6e.png">

BTW, I've had SlowQuitApps running on macOS 12 on my **Intel** MacBook – I have the issue only on the M1 / Apple Silicon Mac.
